hoomd/logging.py

```diff
@@ -367,30 +377,29 @@ def log(func=None,
         func (`method`): class method to make loggable. If using non-default
```

Comment:
An example of using this function would be super helpful for understanding what/how this function can be used. If it is already in the logging tutorial, please point people there with a see also block.

Reply:
I added a link to the tutorial. @b-butler would you prefer to add a working example in the `log` documentation?

Reply:
I can do this.